modules/tensor_mechanics/test/tests/beam/action/tests

```diff
@@ -1,109 +1,174 @@
 [Tests]
+  design = "LineElementMaster/index.md"
+  issues = "#10313"
   [./2_block_action]
     type = 'Exodiff'
     input = '2_block.i'
     exodiff = '2_block_out.e'
+
+    requirement = "The LineElementAction class shall correctly create the beam/truss input blocks"
```

Comment:
The LineElementAction class shall correctly create the objects required for a mechanics simulation using beam or truss elements.